test/e2e/suite/issuers/selfsigned/fixtures.go

```diff
@@ -115,6 +124,15 @@ func generateECCSR() ([]byte, error) {
 	return csr, nil
 }
 
+func generateEd25519Csr() ([]byte, error) {
```

Comment:
nitpick(non-blocking): following the [Google CodeReviewComments guides](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) for initialisms linked to in our [CodingConventions](https://cert-manager.io/docs/contributing/coding-conventions/), this should probably be written as `generateEd25519CSR`

```suggestion
func generateEd25519CSR() ([]byte, error) {
```